README.md

```diff
@@ -3,13 +3,43 @@
 A set of Dockerfiles and a [docker-compose][docker-compose] yml file
 to facilitate the running of the VIP SMS tool.
 
+## Prerequisites
+
+First, you'll need some tools to get the system built:
+
+- Docker: [Docker for Mac][docker] is nice; otherwise install both `docker` and
+  `docker-compose` with your favorite package manager
+- VALID AWS credentials with write permissions to both S3 and SQS
+- [AWS CLI tool][awscli] (also available in Homebrew)
+
+In addition to cloning this repository, you'll need to clone
+the [data-processor repository][data-processor] and
```

Comment:
I believe you're looking for `sms-web` and `sms-worker`, rather than `data-processor` and `metis`.